Dockerfile

```diff
@@ -22,7 +36,8 @@ RUN apk add dumb-init
 # -----------------------------------
 FROM gcr.io/distroless/base-debian11 AS release-default
 
-ARG BIN_NAME
+ARG BIN_NAME=consul-dataplane
```

Comment:
Defaulting these for building locally like we do in [consul-k8s](https://github.com/hashicorp/consul-k8s/blob/8a5eff010a6ee72a72f9e83ea86bc4e52bbf4b2d/control-plane/Dockerfile#L29)